src/cc/tests/CMakeLists.txt

```diff
@@ -58,7 +58,6 @@ foreach(test_dir ${test_dirs})
 endforeach()
 
 set(test_binary test.t)
-message(STATUS "Test sources: ${test_sources}")
```

Comment:
nit: Is there any benefit of limiting the scope of a pull request to changes on the same component? If yes, this commit and a few others should be placed in a separate pull request.